docs/build-on-linea/linea-version/index.mdx

```diff
@@ -11,6 +11,47 @@ import TokenBridgeL2 from "!!raw-loader!/files/testnet/2023-06-13-release/TokenB
 
 Find out all you need to know about the latest Linea versions.
 
+## Alpha v3 release notes
+
+### Summary
+
+The main objective of Alpha v3 is to implement EIP-4844 on Linea, following its introduction to 
+Ethereum mainnet with the Dencun upgrade on March 13.
+
+### Features
+
+#### March 26: EIP-4844
+
+Begin using blob data to post data to L1, significantly reducing data availability costs. 
+Blob data replaces `calldata` and is significantly cheaper.
+
+Multiple contracts have been updated and support blob data. Notable new contract addresses include:
+
+- New verifier contract
+  - Linea Sepolia: [0x5ca5dBf7Cb8F3f3c92E04B16FF5fCA1cdf147f79](https://sepolia.lineascan.build/address/0x5ca5dBf7Cb8F3f3c92E04B16FF5fCA1cdf147f79)
+  - Linea mainnet: TBC
```

Comment:
Should be "M"ainnet to be consistent with Sepolia?

Reply:
Yes, probably. Elsewhere I've always opted for lowercase 'mainnet' (including when talking about Ethereum), but it seems it may be actually be advisable to standardise to uppercase: https://ethereum.org/en/contributing/style-guide/content-standardization/#mainnet